option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message FriendOperatorRequest
{
 repeated U64Type playerid = 1;
}
